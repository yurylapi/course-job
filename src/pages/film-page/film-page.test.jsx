import React from 'react';
import { shallow } from 'enzyme';
import FilmPage from './film-page';

describe('FilmPage component', () => {
    let routeProps;

    beforeAll(() => {
        routeProps = {
            match: {
                params: {
                    filmId: 11111
                }
            }
        };
    });

    it('should be rendered correctly', () => {
        const component = shallow(<FilmPage {...routeProps} />);
        expect(component).toMatchSnapshot();
    });
});
