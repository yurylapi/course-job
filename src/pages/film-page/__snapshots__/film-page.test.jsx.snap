// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmPage component should be rendered correctly 1`] = `
<Fragment>
  <Connect(FilmDetails)
    filmId={11111}
    onCloseFilmHandler={[Function]}
  />
  <withRouter(Connect(SameGenreFilms))
    filmId={11111}
    onOpenFilmHandler={[Function]}
  />
  <Footer />
</Fragment>
`;
